### PR TITLE
[15.0][OU-ADD] sale_coupon_portal_commercial_partner_applicability: Renamed to coupon_portal_commercial_partner_applicability

### PR DESCRIPTION
**Pull request by pilarvargas-tecnativa**
https://github.com/OCA/sale-promotion/pull/139

cc @Tecnativa TT43555

@chienandalu @CarlosRoca13 please review

### CODE REVIEW

**Comment by legalsylvain** [comment]
/ocabot migration coupon_portal_commercial_partner_applicability

**Review by legalsylvain — APPROVED** [review]
code review. LGTM. 
pre-commit error should be fixed. (too long line).

I set "blocked" label, to wait for the merge of the other PR.

**Comment by pilarvargas-tecnativa** [comment]
> code review. LGTM. pre-commit error should be fixed. (too long line).
> 
> I set "blocked" label, to wait for the merge of the other PR.

pre-commit fixed, thanks :)

File: openupgrade_scripts/apriori.py [hunk 1]
```diff
@@ -22,6 +22,7 @@
     "sale_coupon_commercial_partner_applicability": "coupon_commercial_partner_applicability",
     "sale_coupon_mass_mailing": "coupon_mass_mailing",
     "sale_coupon_portal": "coupon_portal",
+    "sale_coupon_portal_commercial_partner_applicability": "sale_coupon_portal_commercial_partner_applicability",  # noqa: B950
```
Comment (chienandalu) [inline openupgrade_scripts/apriori.py]: :slightly_smiling_face:  :point_down: 
```suggestion
    "sale_coupon_portal_commercial_partner_applicability": "coupon_portal_commercial_partner_applicability",  # noqa: B950
```

Comment (pilarvargas-tecnativa) [reply openupgrade_scripts/apriori.py]: I'm going to try but I don't think it will allow it because when you run pre-commit it reformats it in one line.

Comment (chienandalu) [reply openupgrade_scripts/apriori.py]: That's not the fix. Take a closer look :roll_eyes:

**Review by chienandalu — APPROVED** [review]

File: openupgrade_scripts/apriori.py (hunk 1 above)
Comment (pilarvargas-tecnativa) [reply openupgrade_scripts/apriori.py]: But if you include the comment "# noqa: B950" pre-commit will still ignore it in a single line.

Comment (chienandalu) [reply openupgrade_scripts/apriori.py]: My proposal was about the renaming not the pre-commit

Comment (pilarvargas-tecnativa) [reply openupgrade_scripts/apriori.py]: ok, I'm still asleep! I had corrected it but I thought it was for pre-commit, sorry

**Pull request merged**